### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by jkmaswell**
## Summary

Adding footer to project.

## Screenshots

image

## Trello Card

https://trello.com/c/gbX0x57X/14-footer

### CODE REVIEW

File: scss/footer.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  .footer-disc {
+    font-size: $text-big;
+    margin: 0 0 12px;
+    text-transform: uppercase;
+  }
+
+  .footer-copyright {
+    font-size: 11px;
+    font-weight: 300;
+    padding-right: 20px;
```
Comment (FrankIglesias) [inline scss/footer.scss]: use margin instead

File: scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    text-align: right;
```
Comment (FrankIglesias) [inline scss/footer.scss]: is there a flex-box prop that might help you?

File: scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    font-size: $text-big;
+    margin: 0 0 12px;
+    text-transform: uppercase;
+  }
+
+  .footer-copyright {
+    align-self: flex-end;
+    font-size: 11px;
+    font-weight: 300;
+    margin-right: 20px;
```
Comment (FrankIglesias) [inline scss/footer.scss]: why this prop?

Comment (jkmaswell) [reply scss/footer.scss]: to add some space between the screen and the copyright

Comment (FrankIglesias) [reply scss/footer.scss]: Add a padding to the footer

**Pull request merged**